clang/include/clang/ASTMatchers/ASTMatchers.h

```diff
@@ -8288,6 +8288,13 @@ AST_MATCHER_P(Expr, ignoringElidableConstructorCall,
   return InnerMatcher.matches(Node, Finder, Builder);
 }
 
+#ifdef SYCLomatic_CUSTOMIZATION
+AST_MATCHER_P(VarDecl, isSameAs, const VarDecl *, TargetVD) {
```

Comment:
```suggestion
AST_MATCHER_P(DeclRefExpr, isSameAs, const VarDecl *, TargetVD) {
```
Move getdecl and check here

Reply:
done